tests/QS/regtest-nlmo/TEST_FILES

```diff
@@ -0,0 +1,3 @@
+pipek_C6H6.inp                                      85      1e-09           177.2856352706
```

Comment:
How much does the final value (last column) changes in the last step of optimization? If the change is much higher than 1e-09, increase the tolerance from 1e-09 to "the approximate magnitude of the change".